packages/server-wallet/src/__test-with-peers__/wallet/close-channels.test.ts

```diff
@@ -84,8 +87,8 @@ describe('CloseChannels', () => {
       for (const a of aChannels) {
         expect(a.status).toEqual('closed');
       }
-      // Ensure that A has all closed channels
-      const {channelResults: bChannels} = await peerEngines.a.getChannels();
+      // Ensure that B has all closed channels
+      const {channelResults: bChannels} = await peerEngines.b.getChannels();
```

Comment:
What's the context here? It seems like the test has now _changed_. That seems more than just a "clean up" of the closing test.

Reply:
This is a correction of the test assertion.

Previously we were checking if A's channels were closed twice. This change ensures we're checking B's channels are closed as well.